demo_umami.install

```diff
@@ -9,6 +9,29 @@ use Drupal\user\Entity\User;
 use Drupal\user\RoleInterface;
 use Drupal\shortcut\Entity\Shortcut;
 
+/**
+ * Implements hook_requirements().
+ */
+function demo_umami_requirements($phase) {
+  $requirements = [];
+  if ($phase == 'runtime') {
+    $demo_umami_installed_drupal = \Drupal::state()->get('demo_umami_drupal_version');
+
+    $requirements['demo_umami_drupal_version'] = [
+      'title' => 'Demo Umami Drupal Version',
+      'value' => \Drupal::VERSION,
+      'description' => 'Demo Umami is an experimental profile and not to be used in Production(Live) site. Drupal Upgrade is not recomended.',
+      'severity' => REQUIREMENT_WARNING,
+    ];
+
+    // Check if Drupal version has changed since demo_umami was installed.
```

Comment:
We may only want to check for minor updates (i.e. 8.5.x > 8.6.x) rather than just any update (i.e. 8.5.1 -> 8.5.2), as I doubt we would want to (or be able to) introduce breaking changes between small updates.

However we don't know that yet, so for now I'd leave this as is - just something we may want to think about / change in the future.